### ISSUE TITLE
Strange Behavior on Players

### ISSUE THREAD

**Issue by Matynns** [Traben-0/solid_mobs#35]
Hi there! I love this mod, you're doing great work on it!

I've noticed while playing with others (with all players having the mod installed on both client and server), that standing on top of players seems to either fling you in a direction or cause you to bounce up or down. It appears as though a player standing on another player inherits the momentum of the player below them, even after they've stopped moving. For example, a player who stands on the head of a player who just walked south will slide off of their head southwards, or bounce uncontrollably if they've just jumped.

If it's relevant, we are using platform mode. Thank you!

**Comment by Traben-0** [comment]
Does this occur with no other mods loaded?

**Comment by Matynns** [comment]
Yeah, the issue seems to persist even with uninstalling all other mods.
